app/assets/stylesheets/components/_cards.scss

```diff
@@ -0,0 +1,47 @@
+// this has to be moved to container later, put it here because of conflicts
+.arts-container {
+  display: flex;
+  justify-content: space-around;
+  margin-top: 100px;
+  margin-bottom: 20px;
+    a {
+      text-decoration: none;
+      color: white !important;
+    }
+}
```

Comment:
This is part of the index, not the card component. It'll be add in the other issue.

You also added the text-decoration to all the `a` in another PR so you don't need to use that either, just make sure to give the white color to the `.art-card` selector

Reply:
Yes this is just to see, will remove it in last pull request